examples/lang/erc20/src/lib.rs

```diff
@@ -257,7 +258,7 @@ mod tests {
         let bob = AccountId::try_from([0x1; 32]).unwrap();
         assert_eq!(erc20.total_supply(), 1234);
         assert_eq!(erc20.balance_of(alice), 1234);
-        assert_eq!(erc20.transfer_from(alice, bob, 234), true);
```

Comment:
What is bad about `transfer_from` here? I think it is more explicit.

Reply:
Cause then we never test the `transfer` function.

`transfer_from` is tested when you do the allowance stuff

Reply:
I want to actually also add a failed test

Reply:
Sounds fair!

Reply:
With failure test you mean a test that `panics` or a test that returns `false` upon `transfer` ?

Reply:
Just added. A test that tries to transfer more than Alice has, and returns false.